### PR TITLE
stop processing if pod status is not found

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
stop processing if can not found pod status

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by klueska** [comment]
I think this is already addressed in https://github.com/kubernetes/kubernetes/pull/84300 which should be merged soon.

File: pkg/kubelet/cm/cpumanager/cpu_manager.go [hunk 1]
```diff
@@ -353,13 +353,13 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 		allContainers := pod.Spec.InitContainers
 		allContainers = append(allContainers, pod.Spec.Containers...)
 		status, ok := m.podStatusProvider.GetPodStatus(pod.UID)
-		for _, container := range allContainers {
-			if !ok {
-				klog.Warningf("[cpumanager] reconcileState: skipping pod; status not found (pod: %s)", pod.Name)
-				failure = append(failure, reconciledContainer{pod.Name, container.Name, ""})
-				break
-			}
+		if !ok {
```
Comment (mattjmcnaughton) [inline pkg/kubelet/cm/cpumanager/cpu_manager.go]: I agree with you that this execution path reads much cleaner.

However, it does appear that we are using the ability to include `container.Name` in the error. Is there ever a scenario in which including `container.Name` provides a benefit? Given that the pod status can't be found, my guess is no. But I just want to double check.

Comment (mattjmcnaughton) [reply pkg/kubelet/cm/cpumanager/cpu_manager.go]: Whoops, missed @klueska comment :)

**Comment by klueska** [comment]
#84300 was just rebased. Would appreciate some eyes on it when you get the chance.

**Pull request closed without merging**